### PR TITLE
Initial commit BC001 - BU02

### PR DESCRIPTION
**Pull request by pjanck**
- example file
- idsXML
- documentation

### CODE REVIEW

**Comment by pjanck** [comment]
@BenzclyZhang please review as well.

**Comment by evandroAlfieri** [comment]
same for BC01, @BenzclyZhang I did just a small change as a PR to the rail repo. Please have a look

File: tests/BU02/readme.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+- All listed in [TE01](https://github.com/buildingSMART/IFC4.x-IF/tree/main/tests/TE01#ifc-concepts-involved-in-this-test)
+- [Body Geometry](https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/concepts/Product_Shape/Product_Geometric_Representation/Body_Geometry/content.html)
+- [Product Span Positioning](https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/concepts/Object_Connectivity/Product_Span_Positioning/content.html)
+- [Product Relative Positioning](https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/concepts/Object_Connectivity/Product_Span_Positioning/content.html)
+- [Product Linear Placement](https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/concepts/Product_Shape/Product_Placement/Product_Linear_Placement/content.html)
+
+## Prerequisites
+
+The test plan builds upon the [TE01 test plan](../TE01/Readme.md).
```
Comment (BenzclyZhang) [inline tests/BU02/readme.md:35]:
```suggestion
The test plan builds upon the [AL01 test plan](../AL01/Readme.md).
```

File: tests/BU02/readme.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+- The `IfcBuilding` instance representing the station shall be relatively positioned to an `IfcReferent`,
+ which in turn shall be relatively positioned to the `IfcAlignment`.
+- The `IfcBuilding` instance representing the platform shall be span positioned along the `IfcAlignment`.
+- The `IfcReferent` instances shall be linearly placed along the `IfcGradientCurve` representation of the `IfcAlignment`.
+
+### Informal criteria
+
+- Both buildings should lie centrally inside the terrain, "parallel" to the elevated track bed.
+
+!["Perspective view"](./Dataset/skewedview.PNG)
```
Comment (BenzclyZhang) [inline tests/BU02/readme.md:78]:
```suggestion
!["Perspective view"](./Dataset/skewedview.png)
```

**Review by BenzclyZhang — CHANGES_REQUESTED** [review]

**Comment by evandroAlfieri** [comment]
> same for BC01, @BenzclyZhang I did just a small change as a PR to the rail repo. Please have a look

Hi @BenzclyZhang, can you review and approve the small change I suggested - so that this test case can be published? Thanks
cc: @pjanck @apinzenoehler

**Comment by pjanck** [comment]
> same for BC01, @BenzclyZhang I did just a small change as a PR to the rail repo

I don't see a PR anywhere, but I believe I must be looking at the wrong place. Can you provide a link, please?

**Comment by evandroAlfieri** [comment]
Forgot to PR from the patch branch, sorry. Here: https://github.com/bSI-RailwayRoom/IFC4.x-IF/pull/6

**Review by BenzclyZhang — APPROVED** [review]

File: tests/BU02/readme.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+:zap: For this test case to be considered passed, **all criteria listed in this section**, and **the ones of prerequisites tests** shall be verified. :zap:
+
+### Formal rules
+
+#### IFC standard (schema and specification)
+
+When validated using the bSI Validation Service, the IFC must pass:
+
+- Syntax & Schema check
+    - https://github.com/buildingSMART/ifc-gherkin-rules/pull/65
```
Comment (evandroAlfieri) [inline tests/BU02/readme.md:58]: See https://github.com/bSI-RailwayRoom/IFC4.x-IF/pull/7

**Comment by pjanck** [comment]
@evandroAlfieri This should be resolved now and ready for merging.

**Review by evandroAlfieri — APPROVED** [review]

**Pull request merged**